### PR TITLE
Extended HAProxy Frontend and Backend options list

### PR DESCRIPTION
**Pull request by pavel-z1**
This PR has two changes:
1. Added possibility to configure HAProxy repository
2. Extended HAProxy Frontend and Backend options list, some features from HAProxy version 2.

### CODE REVIEW

File: tasks/install-Generic.yml [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+    baseurl:             "{{ item.baseurl }}"
+    gpgcheck:            "{{ item.gpgcheck  | default(omit) }}"
+    gpgkey:              "{{ item.gpgkey  | default(omit) }}"
+    repo_gpgcheck:       "{{ item.repo_gpgcheck | default(omit) }}"
+    file:                "{{ item.file  | default(omit) }}"
+    skip_if_unavailable: "{{ item.skip_if_unavailable | default(omit) }}"
+    priority:            "{{ item.priority | default('99') }}"
+    enabled:             "{{ item.enabled | default(true)}}"
+    state:               "{{ item.state | default('present') }}"
+    proxy:               "{{ zabbix_http_proxy | default(omit) }}"
```
Comment (goldyfruit) [inline tasks/install-Generic.yml]: I guess it should be `item.http_proxy`.

**Comment by goldyfruit** [comment]
@pavel-z1 could you please add some documentation to the README about the `haproxy_repo_yum` option?

File: tasks/install-Generic.yml (hunk 1 above)
Comment (pavel-z1) [reply tasks/install-Generic.yml]: Agree. Fixed.

**Comment by pavel-z1** [comment]
> @pavel-z1 could you please add some documentation to the README about the `haproxy_repo_yum` option?

Hi @goldyfruit , added example to the README.

**Comment by goldyfruit** [comment]
@pavel-z1 you will have to passe the CI tests before been merged, thanks for the updates!

**Comment by goldyfruit** [comment]
I cancel de check by error, but the 3 others were successful. Thanks @pavel-z1 !

**Pull request merged**